include/swift/AST/DiagnosticsSIL.def

```diff
@@ -375,6 +375,12 @@ NOTE(autodiff_nondifferentiable_argument,none,
 NOTE(autodiff_nondifferentiable_result,none,
      "cannot differentiate through a non-differentiable result; do you want to "
      "add '.withoutDerivative()'?", ())
+NOTE(autodiff_noderivative_stored_property,none,
+     "cannot differentiate through a '@noDerivative' stored property; do you "
+     "want to add '.withoutDerivative()'?", ())
+WARNING(autodiff_nonvaried_result_fixit,none,
+        "result does not depend on differentiation arguments and always has a "
```

Comment:
```suggestion
        "result does not depend on differentiation arguments and will always have a "
```

Reply:
Updated in https://github.com/apple/swift/pull/22973/commits/e42706e5b266d61589a06dcb3ee67cfc1a2ac407.